src/commands/update.ts

```diff
@@ -0,0 +1,41 @@
+import { SlashCommandBuilder } from 'discord.js';
+import { exec } from 'node:child_process';
+
+const info = new SlashCommandBuilder()
+	.setName('update')
+	.setDescription('Pulls the latest changes and restarts the bot')
+	.setDefaultMemberPermissions('0');
+
+export const update: GlobalCommand = {
+	info,
+	requiresGuild: false,
+	async execute({ replyPrivately, interaction }) {
+		const admin_ids = process.env['ADMINISTRATORS']?.split(',');
+		if (!admin_ids) {
+			// TODO: make this a UserMessageException
```

Comment:
UserMessageException is part of the EvilHangman PR, can't do this right now without a bunch of unnecessary hassle.